src/FlipCounter.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import "./FlipCounter.css";
+import FlipCountercontainer from "./FlipCounterCONAINER";
+
+interface Props {}
+
+interface State {
+  x: number;
+  value: string;
+}
+
+class FlipCounter extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
```

Comment:
According to [Dan Abramov's component/container idea](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0) component is just a view layer. It does not have any methods, any state, it just renders the data and responsible for the view of the component.

Container is responsible for all the functionality: operating with state, mathematical calculations.

So all these methods should be moved to Container.

App should render Container, which renders Component. Not vice-versa.

Additionally, in the task specification, it is written, that `FlipCounter` component should have only one prop - value. And this FlipCounter does not have any props.